.travis.yml

```diff
@@ -39,3 +39,5 @@ script:
 
 after_success:
   - coveralls
+  - pip install scrutinizer-ocular
```

Comment:
Perhaps for consistency with the coveralls coverage reporting, this could be moved up to the `install` section, after the `pip install python-coveralls`? Anyway, if you think Scrutinizer is useful, I'm happy with adding this PR :+1: